Dockerfile

```diff
@@ -19,11 +20,34 @@ ARG COMMIT
 ARG REPO_INFO
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager -ldflags "-s -w -X github.com/kong/kubernetes-ingress-controller/v2/internal/metadata.Release=$TAG -X github.com/kong/kubernetes-ingress-controller/v2/internal/metadata.Commit=$COMMIT -X github.com/kong/kubernetes-ingress-controller/v2/internal/metadata.Repo=$REPO_INFO" ./internal/cmd/main.go
 
+### FIPS 140-2 binary
+# Build the manager binary
+FROM us-docker.pkg.dev/google.com/api-project-999119582588/go-boringcrypto/golang:1.17.2b7 as builder-fips
```

Comment:
I'm curious to have some extra context on this image as it seems somewhat esoteric?

Reply:
4e5fe8af1caa2f1c9fb4eb373b55af9dd91f035d

Reply:
How would you feel about adding some comments here with the link to the documentation and/or repo that creates this image? I think some kind of breadcrumb here could be helpful for future readers.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/commit/4e5fe8af1caa2f1c9fb4eb373b55af9dd91f035d is that link, unless you're looking for something else? The preview in the review comment didn't update and still shows the version prior to that commit.

Reply:
my bad, thank you